docs/next_link_analysis.md

```diff
@@ -0,0 +1,108 @@
+# Investigating relationships between successive URLs
+
+## Predictable relationships between consecutive URLs 
+
+The hypothesis is that there is no relationship between the sequences of government GOVUK website URLs visited by a user. Clicks are essentially randomly distributed, according to their availability. There is a natural degree of accuracy, of a naive prediction, just due the layout of the government website.
+
+In summary:
+
+- there is a [65%] chance that the next URL will be found in the next [10] recommended URLs
+- the Mean Reciprocal Rank (MRR) for the predictions is [0.43] on a holdout test set
+
+These results should not be regarded in terms of good or bad, rather they just establish a reference score. This serves as a benchmark to be beaten in future tailored predictions, which incorporate additional explanatory data regarding the user. This enables meaningful discussion and development of future models. These measures should be seen as the naiive or random prediction threshold. Any future models should be better than this measure.  
+
+## Data used
+
+The data used in the analysis is pulled from a single hour of usage (midday) of the GovUK website on 2020-11-13. This contains 844k unique records. As can be seen beneath, for a single `fullVisitorId`, there can be multiple values for `visitId`. It relates to hits which are classified as `PAGE`.
```

Comment:
I think the last sentence could be clarified further. Is my suggestion here correct?

```suggestion
The data used in the analysis is pulled from a single hour of usage (midday) of the GovUK website on 2020-11-13. This contains 844k unique records. As can be seen beneath, for a single `fullVisitorId`, there can be multiple values for `visitId`. It relates to the fact that a `visitId` is given to each `pagePath` and user accesses, `hits`, whilst the `fullVisitorId` ties the user across `pagePath`s.
```

Reply:
In the SQL query, the where clause is "hits.type = 'PAGE'"

Reply:
Updated suggestion to make it more explicit.